string action
string param
MementarTimestamp stamp
---
string[] values
MementarTimestamp time_value
int16 code
